project/plugins.sbt

```diff
@@ -10,3 +10,4 @@ addSbtPlugin("com.eed3si9n"              % "sbt-assembly"              % "0.14.1
 addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat"              % "0.1.11")
 addSbtPlugin("ch.epfl.scala"             % "sbt-bloop"                 % "1.4.0-RC1")
 addSbtPlugin("org.jmotor.sbt"            % "sbt-dependency-updates"    % "1.2.2")
+addSbtPlugin("de.heikoseeberger"         % "sbt-header"                % "5.6.0")
```

Comment:
Do we need license headers for some reason? Aesthetically I'm not a big fan of the license headers, and if it's going to be a manual task to update the copyright year on an annual basis anyway (it's still manual right? at least [here](https://github.com/locationtech/geotrellis/blob/master/raster/src/main/scala/geotrellis/raster/equalization/Implicits.scala) the year is still 2019) I'd rather just have that be a small change in the license file

Reply:
It can be good for licensing purposes, and we can overload the task in a such a way that it won't modify old headers (I dont think it makes a lot of sense to update headers for old files; read it as ~ headers are generated once per file). 

Anyway; I'm okay with this plugin removal

Reply:
@jisantuc check this out: https://github.com/azavea/stac4s/pull/73/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR46-R54 

This function does not modify files with old headers;

Reply:
What do you mean "can be good for licensing purposes"? We already have a license file in the root of the repo -- what additional benefit do the headers bring?

Reply:
By that I mean that it is a recommended practice; i.e. following through the ECLIPSE IP checker binaries with headers in all files can automatically pass through their validation system.

^ I don't have hard feelings about adding headers

Reply:
I will remove them from this PR since it was not the idea of this PR at all :D